### PR TITLE
image_partition_action: Add support for FS extended options

### PR DESCRIPTION
**Pull request by cazou**
Extended options can be used for specific file system tuning.

### CODE REVIEW

File: actions/image_partition_action.go [hunk 1]
```diff
@@ -178,18 +178,19 @@ import (
...
+	Name             string
+	PartLabel        string
+	PartType         string
+	PartUUID         string
+	Start            string
+	End              string
+	FS               string
+	Flags            []string
+	Features         []string
+	ExtendedOptions  []string
```
Comment (obbardc) [inline actions/image_partition_action.go:191]: missing documentation

File: actions/image_partition_action.go [hunk 2]
```diff
@@ -178,18 +178,19 @@ import (
...
-	PartType  string
-	PartUUID  string
-	Start     string
-	End       string
-	FS        string
-	Flags     []string
-	Features  []string
-	Fsck      bool "fsck"
-	FSUUID    string
+	number           int
```
Comment (obbardc) [inline actions/image_partition_action.go:181]: nitpick as we haven't formatted the code with gofmt yet; but the formatting is a spurious change; we should only change the line which is relevent

**Review by obbardc — CHANGES_REQUESTED** [review]

File: actions/image_partition_action.go (hunk 2 above)
Comment (cazou) [reply actions/image_partition_action.go:181]: So you'd rather have unaligned types for now ? It won't match the rest of the code. Should I add an additional formatting commit ?

**Comment by cazou** [comment]
@obbardc How is this latest version ?

**Pull request merged**